android/res/values/styles-text.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools">
   <style name="MwmTextAppearance">
-    <item name="android:fontFamily" tools:ignore="NewApi">@string/robotoRegular</item>
+    <item name="android:fontFamily"
```

Comment:
No really need, but it's a maps.me xml code style which was provided by previous developer. Also this xml scheme was shared among rest team members. Thus, I'd propose to leave this new line if it's not really annoying for you, but if you want I will change the formatter scheme and resend to rest team